.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -84,32 +66,45 @@ jobs:
         echo "code: $code"
         echo "$code" | grep -q 200
 
+    - name: Set MANUAL_URL
+      id: set-manual-url
+      run: |
+        if [ "${{ github.event_name }}" == "schedule" ]; then
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        else
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/${{ github.event.pull_request.head.sha }}/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        fi
+
     - uses: ./.github/actions/upload-artifact-with-retry
       with:
         name: nvd
         path: nvd.zip
         if-no-files-found: error
 
-  run-updater:
+  build-and-upload:
     needs:
-    - build-updater
-    - download-nvd
+    - parse-versions
+    - prepare-environment
     runs-on: ubuntu-latest
-    env:
-      STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
-      STACKROX_NVD_API_CALL_INTERVAL: 6s
-
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        vb: ${{ fromJson(needs.parse-versions.outputs.versions) }}
```

Comment:
We might only need the matrix to build... follow me:

1. We run one build step for each version.
2. We put the bundles under directories named after the version `dev/vulnerabilities.zip`, `v1/vulnerabilities.zip`, `v2/vulnerabilities.zip`
3. Upload all bundles to github as artifacts.
4. The upload step (only one) pulls all bundles, and uploads them.

Step (3.) can be done using [upload options](https://github.com/actions/upload-artifact?tab=readme-ov-file#upload-using-multiple-paths-and-exclusions).

Reply:
Why do it?

We don't have to regenerate these artifacts if the upload fails; Very unlikely, but nice to have. And cheap.

Reply:
I have added the step to upload all bundles to artifacts and the upload step (only one) pulls all bundles, and uploads to GCS bucket. Meanwhile I am keeping the matrix step.